config/rbac/controller_roles.yaml

```diff
@@ -0,0 +1,97 @@
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: manager-role
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - get
+  - list
+  - update
+  - watch
+- apiGroups:
+  - adi.io.my.domain
```

Comment:
What is this group? Should this be something more meaningful? Or is this something that someone has already defined elsewhere?

Reply:
definitely should be more meaningful

Reply:
It is being defined in https://github.com/openshift/assisted-service/blob/master/internal/controller/config/rbac/role.yaml#L19. The CRDs are also using adi.io.my.domain: https://github.com/openshift/assisted-service/tree/master/internal/controller/config/crd/bases @filanov Will the apiGroup be renamed sometime in the future?

Reply:
We need to remove the domain. 
ADI stand for agent driven installer. 
I leave the naming to @avishayt and @mhrivnak